internal/rbd/rbd_attach_test.go

```diff
@@ -104,3 +106,62 @@ func TestParseMapOptions(t *testing.T) {
 		})
 	}
 }
+
+func Test_appendCrushLocationMapOptions(t *testing.T) {
+	t.Parallel()
+	type input struct {
+		mapOptions       string
+		crushLocationMap map[string]string
+	}
+	tests := []struct {
+		name string
+		args input
+		want string
+	}{
+		{
+			name: "both empty mapOptions and crushLocationMap",
+			args: input{
+				mapOptions:       "",
+				crushLocationMap: nil,
+			},
+			want: "",
+		},
+		{
+			name: "empty mapOptions and filled crushLocationMap",
+			args: input{
+				mapOptions: "",
+				crushLocationMap: map[string]string{
+					"region": "west",
+				},
+			},
+			want: "read_from_replica=localize,crush_location=region:west",
+		},
+		{
+			name: "filled mapOptions and crushLocationMap",
+			args: input{
+				mapOptions: "read-only=true",
```

Comment:
It doesn't matter for this unit test, but all krbd options use underscores (so `read_only`,  not `read-only`).  Someone grepping for `mapOptions` examples in the future may get confused.

Reply:
https://docs.ceph.com/en/latest/man/8/rbd/#kernel-rbd-krbd-options 

changed it to `notrim` instead to avoid confusion